### ISSUE TITLE
Dropdown / DataTables don't work

### ISSUE THREAD

**Issue by ohaddahan** [dreamingechoes/bootstrap_sb_admin_base_v2#9]
I'm having some issues trying to use this wonderful Gem.
If I include the main stream Bootstrap Gem too the Dropdown doesn't work.

And I wasn't able to get DataTables to work at all :(

Can we get some clear how-to when we want to add other Gems that might have overlap with the data from this one?

Is it possible to have this Gem include only items that aren't included in the main Bootstrap Gem to avoid conflicts?

Thanks!

**Comment by dreamingechoes** [comment]
Hi ohaddahan!

Thank you for take the time to report this issue! I'll take care of this and I'll update the gem ASAP fixing this issue, I'll let you know when all works properly 😄 

Grettings! And sorry for the inconvenience 🙏

**Comment by ohaddahan** [comment]
@dreamingechoes any update?

**Comment by dreamingechoes** [comment]
Hi @ohaddahan!

Apologies for the delay in this issue, the last two weeks I've been very busy at work and I couldn't take care of this issue. Give me a couple more days to solve this and publish a new version, don't worry, I'll let you know when it's done 😄

**Comment by dreamingechoes** [comment]
Hi again @ohaddahan!

Finally this sunday I've took a look at this, but unfortunately I couldn't reproduce this error. I've tested dataTables with the examples they provide on his website, and it works for me (for example, this one: [https://datatables.net/examples/basic_init/zero_configuration.html](https://datatables.net/examples/basic_init/zero_configuration.html)).

Could you provide more information about your case so I can have a better picture of the problem? Rails version, how you handle and create the dataTable (HTML and JS code), specific message error you have... Any detail could be good to have 😄

**Comment by ohaddahan** [comment]
@dreamingechoes I think it was confusion on my part on how to setup application.js / application.css / application.html.erb.

I followed your Sensor Admin demo and now it seems to work fine.
I think that every gem which relies / includes so many generic packages like Bootstrap / jQuery should give very specific instructions on what to do with these files.

For a new project , it's easy but I'm integrating it into a new one and I got a bunch of jQuery / Bootstrap stuff in it. They seem to fight each other.

I would even go so far to make these packages as an explicit dependency instead of including portions of them inside the theme gem.
But that's just my personal view , you did an amazing job!
